timescaledb/how-to-guides/multinode-timescaledb/multinode-maintenance.md

```diff
@@ -23,6 +26,36 @@ $$;
 SELECT add_job('data_node_maintenance', '5m');
 ```
 
+## Maintaining statistics for distributed hypertables
+
+Table statistics need to be up-to-date to allow for efficient planning
+of queries on distributed hypertables. Unfortunately, due to their
+distributed nature, auto-vacuum is not useful to gather the statistics.
+So you need to explicitly ANALYZE your distributed hypertables
+periodically via a maintenance job as shown below.
```

Comment:
```suggestion
On distributed hypertables, the ttable statistics need to be kept updated.
This allows you to efficiently plan your queries. Because of the nature of 
distributed hypertables, you can't use the `auto-vacuum` tool to gather
statistics. Instead, you can explicitly ANALYZE the distributed hypertable
periodically using a maintenance job, like this:
```

Reply:
Reworded